### PR TITLE
DOCS-6035 API Notebook security

### PR DESCRIPTION
**Pull request by hiuxmaycry**

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
Thank you very much @hiuxmaycry ! This message seems good to me. @mcampo and @kuntald , if you agree, I'll polish the formatting and merge.

File: modules/ROOT/pages/to-use-api-notebook.adoc [hunk 1]
```diff
@@ -91,7 +91,14 @@ API.authenticate(github);
 ----
 ====
 
+== API Notebook security
+
+API notebooks run plain JavaScript in the browser. To ensure this does not leak sensitive data, API Notebook disables certain JavaScript functionality:
```
Comment (mcampo) [inline modules/ROOT/pages/to-use-api-notebook.adoc]:
```suggestion
API notebooks run plain JavaScript in the browser. To ensure sensitive data is not leaked, API Notebook disables certain JavaScript functionality:
```

**Review by mcampo — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/to-use-api-notebook.adoc (hunk 1 above)
Comment (hiuxmaycry) [reply modules/ROOT/pages/to-use-api-notebook.adoc]: Done!

**Review by mcampo — APPROVED** [review]
👍

**Comment by wwarner-salesforce** [comment]
Will merge when the code change is deployed.

File: modules/ROOT/pages/to-use-api-notebook.adoc [hunk 2]
```diff
@@ -91,7 +91,14 @@ API.authenticate(github);
 ----
 ====
 
+== API Notebook security
```
Comment (kbostrom) [inline modules/ROOT/pages/to-use-api-notebook.adoc]:
```suggestion
== API Notebook Security
```

File: modules/ROOT/pages/to-use-api-notebook.adoc [hunk 3]
```diff
@@ -91,7 +91,14 @@ API.authenticate(github);
...
+
+API notebooks run plain JavaScript in the browser. To ensure sensitive data does not leak, API Notebook disables certain JavaScript functionality:
+
```
Comment (kbostrom) [inline modules/ROOT/pages/to-use-api-notebook.adoc:97]: You don't need to end the following list items with periods, because they are both fragments.

**Review by kbostrom — APPROVED** [review]
Here you go!

File: modules/ROOT/pages/to-use-api-notebook.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-use-api-notebook.adoc:97]: I'm a little embarrassed to have missed this, and to have missed the capitalization. Thank you very much for fixing them both!

**Pull request merged**